client/a8c-for-agencies/sections/agency-tier/lib/get-agency-tier-info.tsx

```diff
@@ -170,6 +172,52 @@ const getAgencyTierInfo = (
 				learnMoreLink:
 					'https://agencieshelp.automattic.com/knowledge-base/agency-tiering-benefits/#pro-partner',
 			};
+			break;
+		case 'strategic-agency-partner':
+			tierInfo = {
+				title: translate( 'Strategic Agency Partner' ),
+				fullTitle: translate(
+					"{{label}}You're currently a{{/label}} {{title}}Strategic Agency Partner{{/title}}",
+					{
+						components: {
+							label: <div className="agency-tier-overview__current-agency-tier-label"></div>,
+							title: <div className="agency-tier-overview__current-agency-tier-title"></div>,
+						},
+					}
+				),
+				subtitle: preventWidows( translate( "You've reached the highest tier!" ) ),
```

Comment:
We also say this for "Pro Agency Partner". Is that ok?

Reply:
I think it's okay, but @simonktnga8c can confirm. The thing is that if you're on Pro Agency Partner you should not see the new tier as upgrade path so technically the two of them are the highest tiers. But that's a good point and Jeff also brought this up on Slack.